### PR TITLE
ci: update dependencies

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Voron CI
+name: Voron Tap CI
```
Comment (KiloQubit) [inline .github/workflows/ci-build.yml:1]: Please revert the name change here, this job is not specific to Voron Tap, it's the same job as the other repositories.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for you PR!  See some needed changes below.

**Comment by thijstriemstra** [comment]
thanks for the review, reverted name. could you approve the workflow so we can see if it properly runs?

**Review by KiloQubit — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**